dask/dataframe/tests/test_dataframe.py

```diff
@@ -469,20 +465,88 @@ def test_describe(include, exclude, percentiles, subset):
 
     ddf = dd.from_pandas(df, 2)
 
+    if PANDAS_GT_110:
+        datetime_is_numeric = True
+        datetime_is_numeric_kwarg = {"datetime_is_numeric": True}
+    else:
+        datetime_is_numeric = None
+        datetime_is_numeric_kwarg = {}
```

Comment:
Nitpick: we may be able to drop `datetime_is_numeric` here and just use `if datetime_is_numeric_kwarg:` below

Reply:
yeah we can do that - I couldn't decide which was easier to read - if you like the kwarg let's go with that.